content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -6,7 +6,7 @@ content_type: task
 ---
 
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< feature-state for_k8s_version="v1.21" state="GA" >}}
```

Comment:
i don't think this "feature-state" makes a lot of sense here.
`sysctl` is not a tool originating from the Kubernetes project and its usage cannot fall under a GA or Beta state.

Reply:
it was added in this PR:
https://github.com/kubernetes/website/pull/8804

Reply:
So, I should remove it?

I tried to summarize the history of sysctls in https://github.com/kubernetes/enhancements/issues/34#issuecomment-782553906

What we have done in 1.21 is lock the feature gate to True and promote e2e test cases(which is in process).

Reply:
i don't maintain this page, so i was just commenting.
sig-docs may have feedback on this.

Reply:
The feature gates page lists `Sysctls` as a feature. Is that incorrect?